aclk/aclk.c

```diff
@@ -763,8 +763,9 @@ void *aclk_main(void *ptr)
         return NULL;
     }
 
+    unsigned int proto_hdl_cnt;
```

Comment:
Hi Timo! Should this be initialized? Or inside the ifdef below? It is used later on in line 808... ?

Reply:
in such case it is passed but unused
```
void aclk_stats_thread_prepare(int query_thread_count, unsigned int proto_hdl_cnt)
{
#ifndef ENABLE_NEW_CLOUD_PROTOCOL
    UNUSED(proto_hdl_cnt);
#endif
```